internal/graph/check.go

```diff
@@ -257,6 +282,7 @@ func intersection(ctx context.Context, concurrencyLimit uint32, handlers ...Chec
 				continue
 			}
 
+			dbReads += result.resp.GetResolutionMetadata().DatastoreCallCount
 			if !result.resp.GetAllowed() {
 				return result.resp, nil
```

Comment:
Same comment as in https://github.com/openfga/openfga/pull/880/files#r1282110051